data/tutorials/getting-started/1_01_a_tour_of_ocaml.md

```diff
@@ -149,6 +149,16 @@ When `square` is applied to `50`, it evaluates `x * x` into `50 * 50`, which lea
 
 The REPL indicates that the type of `square` is `int -> int`. This means it is a function taking an `int` as a parameter (input) and returning an `int` as result (output). A function value can't be displayed, which is why `<fun>` is printed instead.
 
+```ocaml
+# let square ~x = x * x;;
+val square : x:int -> int = <fun>
+
+# square ~x:50;;
+- : int = 2500
+```
+
+This example defines `square` again but it has a named parameter this time. Function parameters can have labels. Note that the label is included when applying `square`. Labelled parameters are detailed in the [Labelled Arguments](/docs/labels) tutorial.
+
```

Comment:
Maybe add a comment here explaining when labels are useful? For instance to distinguish parameters with the same type?

Reply:
Mentioned functions with several parameters. What do you think?